tests/ui/components/results/verticalresultscomponent.js

```diff
@@ -2,12 +2,23 @@ import DOM from '../../../../src/ui/dom/dom';
 import { mount } from 'enzyme';
 import mockManager from '../../../setup/managermocker';
 import StorageKeys from '../../../../src/core/storage/storagekeys';
+import SearchStates from '../../../../src/core/storage/searchstates';
 import VerticalResultsComponent from '../../../../src/ui/components/results/verticalresultscomponent';
 
+const updateEvents = {};
+
 const mockCore = {
   globalStorage: {
-    set: () => {},
-    on: () => {},
+    set: (storageKey, value) => {
```

Comment:
at this point would it be better to use the real global storage? or were there issues with that

Reply:
that actually worked okay so i'll swap over